### DISCUSSION TITLE
Any plan for porting the ListDetailsView to the Windows Community Toolkit v8.x?

### DISCUSSION THREAD

**Discussion by unliftedq** [CommunityToolkit/Windows#405] [General]
The `ListDetailsView` is quite popular layout in desktop app development, but it's missing in the Windows Community Toolkit v8.x. Although we can implement the function by composing the `SplitView` and `ListView` by developer themselves, it's better to have a Control in the Community Toolkit to reduce such handy work.

Regarding the discussion in the [Windows Community Toolkit v8.0.230907](https://github.com/CommunityToolkit/Windows/discussions/228#discussioncomment-7148570), it didn't get any active response, so I start a new thread here.

**Comment by JoeTomkinson** [comment]
I can't currently see any experiments planned in Labs for this yet so no code to have a play around with. Just adding a comment as a bit of a bump as I think this control was pretty useful. Not sure what the blockers were for migration to v8.x it wasn't actually in hawkers list I dont think when v8 was announced where it outlined controls not ported.

**Comment by JulienTheron** [comment]
I too would like to know why this relatively common control hasn't been ported. It received some love in 7.x where they renamed it from MasterDetailsView so it's a bit curious why it's absent from 8.x.
"Funny" thing is that I found out about this as I was completing the migration to 8.x. I just lost 2 hours.

**Comment by mstasak** [comment]
Aye, same here.  I tried to create a Template Studio WINUI project with master-detail lists, and update it to .NET 8 and this was one of several problems found.  It looks like all versions of CommunityToolkit.WinUI.UI.Controls.Layout got lost, although [Uno.CommunityToolkit.WinUI.UI.Controls.Layout](https://www.nuget.org/packages/Uno.CommunityToolkit.WinUI.UI.Controls.Layout) exists.
